src/components/tool/components/show-example/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Fragment, useEffect, useState } from 'react';
+
+import exampleToString from './utils/example-to-string';
+import { KeyPair } from '../../../../types';
+
+type ShowExampleProps = {
+  fieldName: string;
+  sanityClient: any;
+  typeNames: string[];
+};
+
+const ShowExample = ({ fieldName, sanityClient, typeNames }: ShowExampleProps) => {
+  const [state, setState] = useState(null);
+  useEffect(() => {
+    const query = `*[_type in $typeNames && defined($fieldName)][0]{...}`; // TODO: ideally we get the field off the object inside the query.
```

Comment:
Should we do this then? just need a hand with the groq or something? or was this preexisting?

Reply:
This was pre-existing